frontend/components/ClubEditPage/ClubEditCard.tsx

```diff
@@ -458,24 +458,62 @@ export default function ClubEditCard({
         },
         {
           type: 'content',
-          content: (
-            <>
-              <Text>{FORM_TARGET_DESCRIPTION}</Text>
-              <div className="ml-2 mb-4">
-                <CheckboxLabel>
-                  <Checkbox
-                    checked={showTargetFields}
-                    onChange={(e) => setShowTargetFields(e.target.checked)}
-                    color={BLACK}
-                  />{' '}
-                  <span className="ml-1">
-                    Yes, my {OBJECT_NAME_SINGULAR} is restricted to certain
-                    student groups.
-                  </span>
-                </CheckboxLabel>
-              </div>
-            </>
-          ),
+          content:
+            SITE_ID === 'fyh' ? (
+              <>
+                <Text>{FORM_TARGET_DESCRIPTION}</Text>
+                <div className="ml-2 mb-4">
+                  <CheckboxLabel>
+                    <Checkbox
+                      checked={false}
+                      onChange={(e) => setShowTargetFields(false)}
+                      color={BLACK}
+                    />{' '}
+                    <span className="ml-1">Yes.</span>
+                  </CheckboxLabel>
+                  <CheckboxLabel>
+                    <Checkbox
+                      checked={showTargetFields}
+                      onChange={(e) => setShowTargetFields(true)}
+                      color={BLACK}
+                    />{' '}
+                    <span className="ml-1">
+                      No, my {OBJECT_NAME_SINGULAR} is restricted to certain
+                      student groups.
+                    </span>
+                  </CheckboxLabel>
+                </div>
+                <Text>
+                  If no, Hub@Penn has provided a way for certain student
```

Comment:
This is used by both Penn Clubs and Hub@Penn.

This text should not appear in Penn Clubs. Test with both sites to be sure they look appropriate in both.

Reply:
I think this text only show up in H@P because of the conditional statement at the front

Reply:
It looks good now! I think that `If no` should be `If not`, but refer to their original document for wording.